kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/launch-dev.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+cd ..
+MVN_ARG_LINE=()
+
+for arg in "$@"
+do
+    case "$arg" in
+        *)
+            MVN_ARG_LINE+=("$arg")
+        ;;
+    esac
+done
+
+startDateTime=`date +%s`
+
+# check that Maven args are non empty
+if [ "$MVN_ARG_LINE" != "" ] ; then
+    mvnBin="mvn"
+    if [ -a $M3_HOME/bin/mvn ] ; then
+       mvnBin="$M3_HOME/bin/mvn"
+    fi
+    echo
+    echo "Running maven build on available projects (using Maven binary '$mvnBin')"
+
+    "$mvnBin" -v
+    echo
+    projects=( "MYSERVICE_NAME_MARKER-model" "MYSERVICE_NAME_MARKER-kjar" "MYSERVICE_NAME_MARKER")
+
+    for suffix in "${projects[@]}"; do
+
+        for repository in $suffix;  do
+        echo
+            if [ -d "$repository" ]; then
+                echo "==============================================================================="
+                echo "$repository"
+                echo "==============================================================================="
+
+                cd $repository
+
+                "$mvnBin" "${MVN_ARG_LINE[@]}"
+                returnCode=$?
+
+                if [ $returnCode != 0 ] ; then
+                    exit $returnCode
+                fi
+
+                cd ..
+                fi
+
+        done;
+    done;
+    endDateTime=`date +%s`
+    spentSeconds=`expr $endDateTime - $startDateTime`
+    echo
+    echo "Total build time: ${spentSeconds}s"
+
+else
+    echo "No Maven arguments skipping maven build"
+fi
+
+echo "Launching the application in development mode - requires connection to controller (workbench)"
+pattern="MYSERVICE_NAME_MARKER"
+files=( $pattern )
+cd ${files[0]}
+executable="$(ls  *target//MYSERVICE_NAME_MARKER-MYSERVICE_VERSION_MARKER.jar | tail -n1)"
```

Comment:
better to use 
```
executable="$(ls  *target/*.jar | sort -V | tail -n1)"
```

as this will work regardless of version changes of the service project

Reply:
This failed for me on OSX. Got error when running it so I removed it cause others can run into same issue.